ibis/backends/tests/errors.py

```diff
@@ -65,10 +65,11 @@
     from polars.exceptions import InvalidOperationError as PolarsInvalidOperationError
     from polars.exceptions import PanicException as PolarsPanicException
     from polars.exceptions import SchemaError as PolarsSchemaError
+    from polars.exceptions import SQLInterfaceError as PolarsSQLInterfaceError
 except ImportError:
     PolarsComputeError = PolarsPanicException = PolarsInvalidOperationError = (
         PolarsSchemaError
-    ) = PolarsColumnNotFoundError = None
+    ) = PolarsColumnNotFoundError = PolarsSQLInterfaceError = None
```

Comment:
This is some super surprising formatting.

Reply:
Yeah, chained assignment with a bunch of aliased long-named classes is 🤮

Reply:
OTOH, it's not like this code sees anything other than rote additions and deletions, so optimizing it for aesthetics feels a bit pointless.

Reply:
Yeah, this wasn't a "do something about this comment", more of a "huh, `ruff` formatted that odd" comment.

Reply:
For sure! I'm just having the usual Monday-morning strong opinion :)